packages/arb-token-bridge-ui/src/components/common/HeaderConnectWalletButton.tsx

```diff
@@ -17,8 +19,13 @@ export function HeaderConnectWalletButton() {
     <button
       onClick={showConnectionModal}
       type="button"
-      className="arb-hover rounded-full py-3 text-2xl font-medium text-white lg:bg-lime-dark lg:px-6 lg:text-base lg:font-normal"
+      className="arb-hover flex w-full flex-row items-center rounded-full p-4 text-2xl font-medium text-white lg:bg-lime-dark lg:py-2 lg:px-4 lg:text-base lg:font-normal"
     >
+      <img
+        src={ConnectLogo}
```

Comment:
this icon looks very low res
<img width="222" alt="image" src="https://user-images.githubusercontent.com/13184582/216050061-9e61119d-e736-4af0-bd5c-053e045493cd.png">
can't we use hero icon's plus circle instead?

Reply:
<img width="202" alt="image" src="https://user-images.githubusercontent.com/13184582/217234454-5d6badb8-d29d-4046-9c04-fccb198cd7fe.png">
i see that the icon is using text black now, is this the colour we want?